packages/perseus/src/article-renderer.tsx

```diff
@@ -34,6 +34,9 @@ class ArticleRenderer extends React.Component<any, any> {
         ]).isRequired,
 
         // Whether to use the new Bibliotron styles for articles
+        /**
+         * @deprecated Does nothing
```

Comment:
@jeremywiebe I am just not familiar enough with Perseus to know how to remove this without causing errors to throw. I assumed it would be called by the consumer somehow?

Reply:
I figured that would be a breaking change to the external API. Please let me know if I'm mistaklen.

Reply:
It would be. But if we're doing this work, it'd be good to clean up this (now dead) flag.  In webapp and mobile, we always set the value to `true` so it'd just be a matter of deleting those props.

Reply:
<img width="762" alt="image" src="https://github.com/Khan/perseus/assets/77138/973bb33a-f16a-49c8-b4b1-9c87368292a5">

Reply:
Ok-- well, we'd have to do that in a follow-up task, and we can't remove it here until it's been removed from those two places, right?

Reply:
Or I guess it could be part of a deploy but that's leaving it down to whomever cuts the next release to know to do that or break articles everywhere

Reply:
I'd rather it all happen at once. We have a history of making cleanup tasks and leaving them undone.  If you land this change, you can also cut a Perseus release and then follow up by updating webapp with the new version and the accompanying changes.  I'm ok if Mobile is left for now as it's a super tiny change and will be very easy to adjust when we upgrade Perseus there.